src/static/css/cf-enhancements.less

```diff
@@ -972,6 +972,10 @@ textarea.input__long {
 .content__half-top-on-desk {
     .respond-to-min(800px, {
         padding-top: 20px;
+
+        &:after {
+          top: 20px!important;
```

Comment:
I've usually seen `!important` preceded by a space, will the lack of space here cause issues?